server/src/com/cloud/network/router/CommandSetupHelper.java

```diff
@@ -848,13 +849,38 @@ public int compare(final PublicIpAddress o1, final PublicIpAddress o2) {
                 associatedWithNetworkId = ipAddrList.get(0).getNetworkId();
             }
 
+            // for network if the ips does not have any rules, then only last ip
+            List<IPAddressVO> userIps = _ipAddressDao.listByAssociatedNetwork(associatedWithNetworkId, null);
+
+            int ipsWithrules = 0;
+            int ipsStaticNat = 0;
+            for (IPAddressVO ip : userIps) {
+                if ( _rulesDao.countRulesByIpIdAndState(ip.getId(), FirewallRule.State.Active) > 0 ) {
```

Comment:
Inconsistent formatting of the parentheses in the `if`. A bit nitpicky, but the code has to adhere to format.